elements/pfe-sass/maps/_typography_deprecated.scss

```diff
@@ -0,0 +1,35 @@
+
+// I dont think we want to set values of actual vars, we want them to remain empty
+// leaving these as a map instead 
+
```

Comment:
I don't think we want to set values of actual vars, we want them to remain empty,  leaving these as a map instead

Reply:
More notes about variables: the only want to achieve the re-sizing of type on mobile is through the use of variables. Its a separate file though, pfe-typography-mobile.css